doc/rst/users-guide/base/basicTypes.rst

```diff
@@ -10,7 +10,7 @@ numbers, and strings.  Specifically, these types provide support for:
 
 * Boolean values;
 * Signed and unsigned integers;
-* Real, imaginary, and complex floating point values; and
+* Real, imaginary, and complex floating-point values; and
```

Comment:
From a quick googling, it looks like the two versions are used almost interchangeably, so I'd rather not make any changes to which version is used.